### PR TITLE
tstore tests 8,9,11,12,13

### PR DESCRIPTION
**Pull request by ewilz**

### CODE REVIEW

**Comment by ewilz** [comment]
hmm not sure why the json for 8 and 5 changed? I guess I ran them locally with something different in my env?

File: src/GeneralStateTestsFiller/stExample/09_revertUndoesAllFiller.yml [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# 04
```
Comment (snreynolds) [inline src/GeneralStateTestsFiller/stExample/09_revertUndoesAllFiller.yml]: Can you update the comments at the top? I write the test number, then description, then what we should expect in storage.

Comment (snreynolds) [reply src/GeneralStateTestsFiller/stExample/09_revertUndoesAllFiller.yml]: same for tests 11, 12, & 13!

**Review by snreynolds — CHANGES_REQUESTED** [review]

**Comment by snreynolds** [comment]
> hmm not sure why the json for 8 and 5 changed? I guess I ran them locally with something different in my env?

yeah its fine, I'll re-run them all anyways

File: src/GeneralStateTestsFiller/stExample/09_revertUndoesAllFiller.yml (hunk 1 above)
Comment (ewilz) [reply src/GeneralStateTestsFiller/stExample/09_revertUndoesAllFiller.yml]: woops never read the comments at the top

Comment (ewilz) [reply src/GeneralStateTestsFiller/stExample/09_revertUndoesAllFiller.yml]: thought it was boilerplate

**Pull request merged**